parser/scanner.l

```diff
@@ -850,6 +852,10 @@ year_month		{y}{e}{a}{r}_{m}{o}{n}{t}{h}
 {outer}			return outer
 {password}		lval.item = string(l.val)
 			return password
+{pow}			lval.item = string(l.val)
```

Comment:
pow and power are not reserved keywords in MySQL. So they can be used as identifiers. You need to add them to the rule of NotKeywordToken. Then they can be used as identifiers.
https://github.com/pingcap/tidb/blob/master/parser/parser.y#L1685